rocky/tests/test_boefjes_tasks.py

```diff
@@ -91,6 +94,34 @@ def test_reschedule_task(rf, client_member, mocker, task):
     )
 
 
+def test_reschedule_task_from_other_org(rf, client_member, client_member_b, mocker, task):
+    mock_scheduler_client = mocker.patch("rocky.views.tasks.client")
+    mock_scheduler_client.get_task_details.return_value = task
+    mocker.patch("tools.view_helpers.client.session")
+    mocker.MagicMock()
```

Comment:
Calling this doesn't have any effect?

Reply:
You are right, this was redundant. Removed!